src/tools/quemap/fog.c

```diff
@@ -66,7 +66,12 @@ static void FogForEntity(const cm_entity_t *entity) {
 		if (Cm_EntityValue(entity, "fog_absorption")->parsed ||
 			Cm_EntityValue(entity, "fog_color")->parsed ||
 			Cm_EntityValue(entity, "fog_density")->parsed ||
-			Cm_EntityValue(entity, "fog_noise")->parsed) {
+			Cm_EntityValue(entity, "fog_noise")->parsed ||
+			Cm_EntityValue(entity, "fog_frequency")->parsed ||
+			Cm_EntityValue(entity, "fog_amplitude")->parsed ||
+			Cm_EntityValue(entity, "fog_lacunarity")->parsed ||
```

Comment:
I've never heard this word before.

Side note, you like the entity parsing stuffs, doncha? :)